jobis-infrastructure/src/main/java/team/retum/jobis/domain/banner/presentation/dto/CreateBannerWebRequest.java

```diff
@@ -0,0 +1,36 @@
+package team.retum.jobis.domain.banner.presentation.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import team.retum.jobis.domain.banner.dto.CreateBannerRequest;
+import team.retum.jobis.domain.banner.model.BannerType;
+
+import java.time.LocalDate;
+
+@Getter
+@NoArgsConstructor
+public class CreateBannerWebRequest {
+
+    @NotNull
+    private String bannerUrl;
+
+    @NotNull
+    private BannerType bannerType;
+
+    @NotBlank
+    private LocalDate startDate;
+
+    @NotBlank
+    private LocalDate endDate;

```

Comment:
```suggestion
    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
    private LocalDate startDate;

    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
    private LocalDate endDate;
```
날짜 형식 지정해주면 좋을 것 같아요